src/components/Tooltip/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './tooltip';
```

Comment:
When I tried to add a story it's throwing an error. You can create a simple story by looking at https://github.com/CBIIT/bento-tools/blob/master/src/stories/index.js#L16 you just need to call the component in the story will create a new story for the tooltip

![image](https://user-images.githubusercontent.com/26462504/121029884-f840d980-c776-11eb-8a9b-24967aff307e.png)